spacialgaze-plugins/misc.js

```diff
@@ -433,4 +433,21 @@ exports.commands = {
 		Db.disabledScrolls.remove(target);
 	},
 	enableintroscrollhelp: ["/enableintroscroll [room] - Enables scroll bar preset in the room's roomintro."],
+
+	pmroom: 'rmall',
+	roompm: 'rmall',
+	rmall: function (target, room, user) {
+		if (!this.can('declare', null, room)) return this.errorReply("/rmall - Access denied.");
+		if (room.id === 'lobby') return this.errorReply("This command cannot be used in Lobby.");
+		if (!target) return this.sendReply("/rmall [message] - Sends a pm to all users in the room.");
+		target = target.replace(/<(?:.|\n)*?>/gm, '');
+
+		let pmName = ' SG Server';
```

Comment:
@Lord-Haji, the extra space is actually the user's rank. When messages show up in PMs, the protocol is like this:

`|pm|<rank><sender>|<rank><receiver>|<message>`

If the sender doesn't have a valid global rank, a space is put there and is interpreted as a regular user.

So, yes, the space is actually needed. Please learn the protocol before saying stuff like that